src/app/domain/osReceive/models/OsReceiveCozyApp.ts

```diff
@@ -0,0 +1,21 @@
+export interface OsReceiveCozyApp {
```

Comment:
yes my aim was not to add full app typings but only sharing app properties that I use. Also as you can see the attributes seem somehow duplicated, that's how it was on my local stack, so to be on the safe side I assumed in runtime that it could be both